src/customer/strategies/amazon-maxo/amazon-maxo-customer-strategy.ts

```diff
@@ -6,7 +6,7 @@ import { CustomerInitializeOptions, CustomerRequestOptions } from '../../custome
 import CustomerStrategy from '../customer-strategy';
 
 export default class AmazonMaxoCustomerStrategy implements CustomerStrategy {
-    private _methodId?: string;
+    private _methodId!: string;
```

Comment:
This prevents an null value or undefined, but as I remember Sydney team don't like this, and they recommend us to use `?` and when we need to use we can keep a getter method, you can keep this but I almost sure Sydney team request you to remove it 🍹

Reply:
I have seen this used in quite some sections of the code on  the sdk code, I think I'll leave it here and discuss the use of the exclamation mark with the Sidney Team ♻️